### ISSUE TITLE
README should indicate that pmca-gui works on Linux without any problems

### ISSUE THREAD

**Issue by kureta** [ma1co/Sony-PMCA-RE#123]
I saw that the tool is written in Python so I gave it a shot on my desktop. It worked flawlessly, much better than on MacOS version at least. Just a couple of lines can be added to the README somewhere to let people know they can use this on Linux too. For example:

We do not officially support Linux but you can try to run the pmca-gui. Just clone the repository and run the following commands in the source code directory:

    pip install --requirement requirements.txt
    python pmca-gui.py

You can even create a pip package if you have time.

Thanks!

**Comment by solarkraft** [comment]
... Except it didn't work right away for me on Manjaro: It first complains there is no `fastcache` (quickly solved with a `pip3 install fastcache`), but it also requires `libtk8.6.so` (`ImportError: libtk8.6.so: cannot open shared object file: No such file or directory`), which needs to be installed using pacman ( `sudo pacman -S tk`).

**Comment by satcom886** [comment]
Yes, it should be also mentioned, that you should install the dependencies.

**Comment by nrnrnr** [comment]
The `requirements.txt` contains some lines `sys_platform = 'win32'`.  What is the correct `sys_platform` for Linux?
